tests/unit/resources/test_omnistack_clusters.py

```diff
@@ -90,6 +91,17 @@ def test_get_by_data(self):
         self.assertIsInstance(obj, clusters.OmnistackCluster)
         self.assertEqual(obj.data, resource_data)
 
+    @mock.patch.object(Connection, "get")
+    def test_get_connected_clusters(self, mock_get):
+        mock_get.side_effect = [{'omnistack_clusters': [{'id': '12345'}]}, {'omnistack_clusters': [{'id': '12345'}]}]
+
+        cluster_data = {'name': 'name1', 'id': '12345'}
+        cluster = self.clusters.get_by_data(cluster_data)
+        cluster.get_connected_clusters()
```

Comment:
assigned the return value of get_connected_clusters() to a variable, and check the instance type of the objects returned.

Reply:
Sure will update as per suggestion.

Reply:
Has updated the code to check returned objects.